nodejs/awsx/ec2/vpc.ts

```diff
@@ -71,12 +73,12 @@ export class Vpc extends pulumi.ComponentResource {
                 enableDnsHostnames: utils.ifUndefined(args.enableDnsHostnames, true),
                 enableDnsSupport: utils.ifUndefined(args.enableDnsSupport, true),
                 instanceTenancy: utils.ifUndefined(args.instanceTenancy, "default"),
-            });
+            }, parentOpts);
```

Comment:
this was the root issue.  we were creating the underlying resource without properly parenting it to the awsx.ec2.Vpc.